iexec-core-library/src/test/java/com/iexec/core/task/TaskStatusTests.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2023-2023 IEXEC BLOCKCHAIN TECH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.iexec.core.task;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class TaskStatusTests {
+
+    // region contribution phase
+    @ParameterizedTest
+    @EnumSource(value = TaskStatus.class, names = {"INITIALIZED", "RUNNING"})
+    void isInContributionPhase(TaskStatus taskStatus) {
+        assertTrue(TaskStatus.isInContributionPhase(taskStatus));
+    }
+
+    @ParameterizedTest
+    @EnumSource(value = TaskStatus.class, names = {"INITIALIZED", "RUNNING"}, mode = EnumSource.Mode.EXCLUDE)
```

Comment:
For clarity purpose, should we reorder params here? I'd like to see `mode` before `names`. That would make it easier to read: "Ok, we'll use TaskStatus values. The following values are excluded: INITIALIZED, RUNNING. Got it!" instead of: "Ok, we'll use TaskStatus values. These are INITIALIZED, RUNNING. Got it! Oh, no, they are excluded."
Not a big deal, but would be great IMHO :)

Reply:
see f433fbb